Tools/Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py

```diff
@@ -37,14 +37,14 @@
 from webkitcorepy.task_pool import TaskPool
 from webkitcorepy.terminal import Terminal
 from webkitcorepy.environment import Environment
-from webkitcorepy.credentials import credentials
+from webkitcorepy.credentials import credentials, delete_credentials
```

Comment:
is this new import needed?

Reply:
This is more accurately an "export." This allows us to do `from webkitcorepy import delete_credentials` instead of `from webkitcorepy.credentials import delete_credentials`